assembly/assembly-main/src/assembly/tomcat/conf/server.xml

```diff
@@ -140,8 +140,12 @@
         <!--
         <Valve className="org.apache.catalina.authenticator.SingleSignOn" />
         -->
-         <Valve className="org.apache.catalina.valves.RemoteIpValve" protocolHeader="X-Forwarded-Proto"
-                 internalProxies="\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}"/>
+
+        <Valve className="org.apache.catalina.valves.RemoteIpValve"
+           remoteIpHeader="x-forwarded-for"
+           remoteIpProxiesHeader="x-forwarded-by"
+           protocolHeader="x-forwarded-proto"
+    	  />
```

Comment:
@skabashnyuk I am a bit confused by the tomcat docs and not sure if the old version does the same thing. do you know which headers are needed for proper reverse proxying?

Reply:
>do you know which headers are needed for proper reverse proxying

I think ```X-Forwarded-Proto```

Question: why did you change server.xml?

Reply:
I think value ```internalProxies``` is mandatory and has to be added back.

Reply:
@skabashnyuk thanks for the feedback, i will test again with the old values and see if it still works.